Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Issue.hs

```diff
@@ -186,13 +204,67 @@ createIssueReport (driverId, _) Common.IssueReportReq {..} = do
             categoryId = cast categoryId,
             mediaFiles = cast <$> mediaFiles,
             assignee = Nothing,
-            status = D.NEW,
+            status = D.OPEN,
             deleted = False,
+            ticketId = Nothing,
             createdAt = now,
             updatedAt = now,
             description
           }
 
+    mkTicket :: D.IssueReport -> D.IssueCategory -> Maybe D.IssueOption -> Maybe Ride.Ride -> ShortId DM.Merchant -> [Text] -> Flow Kapture.CreateTicketReq
+    mkTicket issue category mbOption mbRide shortId mediaFileUrls = do
+      info <- mkRideInfo mbRide shortId
+      person <- QP.findById driverId >>= fromMaybeM (PersonNotFound driverId.getId)
+      return $
+        Kapture.CreateTicketReq
+          { category = category.category,
+            subCategory = (.option) <$> mbOption,
+            issueId = Just issue.id.getId,
+            issueDescription = description,
+            mediaFiles = Just mediaFileUrls,
+            name = Just $ person.firstName <> " " <> fromMaybe "" person.lastName,
+            phoneNo = person.unencryptedMobileNumber,
+            personId = person.id.getId,
+            classification = Kapture.DRIVER,
+            rideDescription = info
+          }
+
+    mkRideInfo mbRide shortId = case mbRide of
```

Comment:
It is strange when `mkRideInfo` return `Maybe` type. So it can be refactored using `forM`:
```
    buildTicket :: D.IssueReport -> D.IssueCategory -> Maybe D.IssueOption -> Maybe Ride.Ride -> ShortId DM.Merchant -> [Text] -> Flow Kapture.CreateTicketReq
    buildTicket issue category mbOption mbRide shortId mediaFileUrls = do
      info <- forM mbRide (buildRideInfo shortId)
     ......................

    buildRideInfo shortId ride = do
      res <- DRide.rideInfo shortId (cast ride.id)
      return
        Kapture.RideInfo
          { rideShortId = Just shortId.getShortId,
           ...............
```
By the way we usually use build in names for monadic functions, and mk for pure functions